pkgs/development/python-modules/snmpsim/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pysnmp,
+  poetry-core,
+  pytestCheckHook,
+  pytest-asyncio,
+}:
+
+buildPythonPackage rec {
+  pname = "snmpsim";
+  version = "1.1.7";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "lextudio";
+    repo = "snmpsim";
+    rev = "v${version}";
+    hash = "sha256-ku2u6/sMGf4P716B9qlhCNerslZ3txQh7V+Bq7DUvVk=";
+  };
+
+  build-system = [ poetry-core ];
+
+  dependencies = [ pysnmp ];
+
```

Comment:
Could you enable `pythonImportsCheck`?

Reply:
Yes, but I do not know what module to import. There is no root module. Should I import-check every submodule?